### ISSUE TITLE
Homebrew distribution

### ISSUE THREAD

**Issue by koliyo** [google/magika#672]
Please add a homebrew cask for this tool

**Comment by reyammer** [comment]
Indeed, that would be great. Marked as help needed: in case someone fluent with homebrew has time, help is very welcome on this one. Please feel free to reach out and we can discuss the details (we are about to release a new python module + rust client, so some details may change, it's good to sync first).

**Comment by reyammer** [comment]
Turns out someone took care of it: https://formulae.brew.sh/formula/magika (thanks!)
